go/atreugo/go.sum

```diff
@@ -0,0 +1,37 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
could you remove this file ? no need here since we do not have to keep the exact dependency tree for various deployment